gen.sh

```diff
@@ -1,67 +1,32 @@
 #!/bin/bash
-# Script to generate the Protobuf classes from googleapis/google-cloudevents
 set -e
 
-echo "~ START"
-PROTOBUF_VERSION=3.12.3
+EVENTS_SPEC_REPO='https://github.com/michaelawyu/google-cloudevents'
```

Comment:
Use the official repo or add a TODO.

Reply:
The TO-DO has been listed in the PR description.
Done.